### DISCUSSION TITLE
Error - No installer found for Google.CloudSDK

### DISCUSSION THREAD

**Discussion by sarmentox** [svrooij/WinTuner#150] [Q&A] [answered]
Hi all

I come across certain apps that are available via winget but they are unable to be packaged via via Wintuner. The latest example is Google.CloudSDK
![image](https://github.com/user-attachments/assets/19b7e069-3057-4fc0-9ec9-83bc9ebebd42)

I am able to package other apps without any issues. I confirmed the installer URL from Google works.
Any ideas?

**Comment by svrooij** [comment, marked as answer]
You seem to be using the old cli, can you try to switch to the PowerShell module https://wintuner.app/docs/category/wintuner-powershell
